### PR TITLE
Update create form

### LINKED ISSUE #894: Styleguide: Forms: Layout
Document Form Layout used within the app in a styleguide section

This includes:

- Header ( If present )
- Header Title
- Form Group
- Form Group Title
- Label Alignment
- Buttons ( Inline forms vs. modal forms )
- Button alignment cancel vs submit

**Refactor/cleaup form layouts where necessary.**

Add:

- `<Section name="form-layout" title="Form Layout"></Section>`
- define a `FormLayout.scss`
- define a `FormLayout.js` in `/styleguide/components/sections`

Styleguide Index: #884
reference #877

### PR DESCRIPTION
**Pull request by mparke**
Updates Add Linode form layout and closes #894 ( layout basics already added to styleguide index).

### CODE REVIEW

File: scss/components/Header.scss [hunk 1]
```diff
@@ -109,10 +109,11 @@ $header-padding: $grid-gutter-width-base / 2;
...
     &-badge {
         @extend .badge;
+        background: $red;
+        border: 4px solid $dark-black;
         position: absolute;
-        top: 2px;
         right: 0;
+        top: 2px;
         line-height: 22px;
-        border: 4px solid $dark-black;
```
Comment (eatonphil) [inline scss/components/Header.scss]: Why the reordering?

Comment (mparke) [inline scss/components/Header.scss]: habit of alphabetical ordering. can change back if there's a pattern already in place.

Comment (eatonphil) [inline scss/components/Header.scss]: Nope, no pattern in place. I thought it might be alphabetical here but you did not reorder the line-height.

**Comment by briansteffens** [comment]
Does this change the visuals or behavior of the page? Or is it just cleaning up the CSS and using the newer components?

**Comment by eatonphil** [comment]
Just cleanup

**Pull request merged**